app/javascript/packs/unpublishPostModal.jsx

```diff
@@ -30,127 +28,60 @@ async function confirmAdminUnpublishPost(id, username, slug) {
     });
   }
 
-  toggleUnpublishPostModal();
+  closeModal();
```

Comment:
nitpick / not blocking - since we only use `closeModal` once, we could just inline here `closeWindowModal(window.parent.document)` and avoid needing a separate function for it

Reply:
Applied these changes. Thanks.